receiver/ciscoosreceiver/config.go

```diff
@@ -0,0 +1,87 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ciscoosreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/ciscoosreceiver"
+
+import (
+	"errors"
+	"time"
+
+	"go.opentelemetry.io/collector/scraper/scraperhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/ciscoosreceiver/internal/metadata"
+)
+
+// Config represents the receiver configuration
+type Config struct {
+	scraperhelper.ControllerConfig `mapstructure:",squash"`
+	metadata.MetricsBuilderConfig  `mapstructure:",squash"`
+
+	CollectionInterval time.Duration  `mapstructure:"collection_interval"`
+	Devices            []DeviceConfig `mapstructure:"devices"`
+	Timeout            time.Duration  `mapstructure:"timeout"`
```

Comment:
`scraperhelper.ControllerConfig` already has `CollectionInterval` and `Timeout` fields. Let's not duplicate them